README.md

```diff
@@ -1 +1,64 @@
 # nextcloud-shibboleth-sync
+
+This repo contains an app to synchronize a list of identity providers with the 
+[user_saml](https://github.com/nextcloud/user_saml) Nextcloud application.
+
+This app is written in pure php without dependency and connect to a postgresql database.
+
+## How to build the image
+
+The docker image will copy the `app` directory and the `CMD` instruction directly executes the script.
```

Comment:
You should maybe say that the base image is our home made one? And add a link to it?

Reply:
No it's the `php` image.